Markup kadai 07

blue2wo opened the pull request:
練習課題7「マークアップの思考トレーニング」1st commit
a251bd9 HTML CSS

piasoratech commented on markup_kadai_07/markup_kadai_07.html:
```diff
@@ -0,0 +1,80 @@
+        <h2>スケジュール</h2>
+        <table>
+            <tr>
+                <td style="text-align: center"><b>11:00</b></td>
```
`style`はすべてCSSで指定しましょう。
また、課題では「CSSファイルを編集しない」ように書いてありますが、時間部分を太字にするのも含めてCSSを編集して実現してみましょう

piasoratech commented on markup_kadai_07/markup_kadai_07.html:
```diff
@@ -0,0 +1,80 @@
+        <p>百野：080-0000-0000<br>
+            川﨑：080-0000-0000<br>
+            亀田：080-0000-0000<br>
+            会社代表電話：010-0000-0000</p>
```
ここは各行に`p`タグを使用しましょう

blue2wo commented:
8d7fe9a
- removed unnecessary code and other kadai files. Only kadai 07 files remain.
- removed `<td style="text-align: center"><b>11:00</b></td>`, added to style sheet as requested
- changed `<br>` tags to `<p>` tags

piasoratech commented on markup_kadai_07/markup_kadai_07.html:
```diff
@@ -0,0 +1,80 @@
+            東京都渋谷区恵比寿4-20-4<br>
+            恵比寿ガーデンプレイス グラススクエアB1<br>
+            「QUELCODE 恵比寿ガーデンプレイス校」<br></p>
+        <p> 初めて教室に来られる方は<a href="#">こちら</a>をご参考ください。<br>（当日はノートに いてある待ち合わせ場所ではなく、ノート一番下に記載ある教室まで直接お越しくださいませ）</p> 
```
ここの()内の字が消えてしまってますかね…？

blue2wo commented:
練習課題7「マークアップの思考トレーニング」3rd push - feedback changes

35085f3
- changed remaining <br> tags to <p> tags

blue2wo commented:
練習課題7「マークアップの思考トレーニング」3rd push - feedback changes

35085f3
- changed the remaining `<br>` tags to `<p>` tags

fjtawork commented:
良いかと！
